src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVelesTracking.java

```diff
@@ -27,34 +34,101 @@
 
   public static final String VELES_TRACKING_PRICE_PARAMETER = "bidder_8";
 
+  private void testVeles(String urlFragment, Map<String, String> positionsAndPrices) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "testVeles" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)